modules/site_studio_config_management/tests/src/Kernel/DrushCommandTraitTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Drupal\Tests\site_studio_config_management\Kernel;
+
+use Consolidation\SiteAlias\SiteAlias;
+use Consolidation\SiteAlias\SiteAliasManager;
+use Consolidation\SiteProcess\ProcessBase;
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\site_studio_config_management\Traits\DrushCommandTrait;
+use Drush\Drush;
+use Drush\SiteAlias\ProcessManager;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Path;
+
+class DrushCommandTraitTest extends KernelTestBase {
+
+  use DrushCommandTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = ["site_studio_config_management"];
+
+  /**
+   * The mock object for logger service.
+   *
+   * @var mixed|\PHPUnit\Framework\MockObject\MockObject|LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp(): void {
+    $this->logger = $this->createMock('\Psr\Log\LoggerInterface');
+    $this->logger->expects($this->any())
+      ->method('error')
+      ->willReturnCallback(
+        function ($message) {
+          $this->assertStringContainsString('Command "something" is not defined.', $message);
```

Comment:
1 of the isolated test is failing here @vishalkhode1

Reply:
Yes, fixed in local. Will push changes along with other.

Reply:
Fixed.